include/boost/json/string.hpp

```diff
@@ -1082,18 +1173,20 @@ class string
     //
     //------------------------------------------------------
 
-    /** Access a character, with bounds checking
+    /** Access a character with bounds checking.
 
         Returns a reference to the character specified at
-        location `pos`, with bounds checking. If pos is not
-        within the range of the container, an exception of
-        type `std::out_of_range` is thrown.
+        location `pos`.
 
         @par Complexity
 
         Constant.
 
-        @param pos A zero-based index
+        @par Exception Safety
+
+        Strong guarantee.
+
+        @param pos A zero-based index to access
```

Comment:
Always use punctuation (a period here).

I would just say "A zero based character index." I would avoid saying "to access" because then the text is less copy-patable. For example, you can't say "A zero-based index to access" for the function `insert` that takes a pos, but you can say "A zero based character index."